aten/src/ATen/DLConvertor.h

```diff
@@ -13,8 +13,11 @@ namespace at {
 TORCH_API ScalarType toScalarType(const DLDataType& dtype);
 TORCH_API DLManagedTensor* toDLPack(const Tensor& src);
 TORCH_API Tensor fromDLPack(DLManagedTensor* src);
+TORCH_API Tensor fromDLPack(const DLManagedTensor* src);
```

Comment:
As those functions are semantically incorrect, let's mark them as deprecated an implement as headers to casts to a const variants
```suggestion
C10_DEPRECATED_MESSAGE("Please migrate to a non-const variant")
inline Tensor fromDLPack(const DLManagedTensor* src) { return fromDLPack(const_cast< DLManagedTensor*>(src); }
```

Reply:
Thanks, I have modified just the header file as you suggested, which is minimal changes and makes sense to me. I have started a run on my end to verify it works. Will confirm it by EOD.